models/userManagement.js

```diff
@@ -0,0 +1,100 @@
+const { query } = require('./model');
```

Comment:
I think this was from the old main branch, everything has been moved to userController. 
We can remove this entirely :)